src/components/StripCard/StripCard.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import styles from './StripCard.st.css';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+
+export interface StripCardProps {
+  media?: React.ReactNode;
+  info?: React.ReactNode;
+  roundMedia?: boolean;
+  withoutSidePadding?: boolean;
+}
+
+interface DefaultProps {
+  roundMedia: boolean;
+  withoutSidePadding: boolean;
+}
+
+export class StripCard extends React.Component<StripCardProps> {
+  static displayName = 'StripCard';
+  static defaultProps: DefaultProps = {
+    roundMedia: false,
+    withoutSidePadding: false,
+  };
+
+  render() {
+    const { media, info, roundMedia, withoutSidePadding, ...rest } = this.props;
+
+    return (
+      <TPAComponentsConsumer>
+        {({ mobile }) => (
+          <div
+            {...styles(
+              'root',
+              { mobile, roundMedia, withoutSidePadding },
+              rest,
+            )}
+          >
+            {media && <div className={styles.mediaContainer}>{media}</div>}
```

Comment:
I prefer using a ternary operator:
```
{media ? <div className={styles.mediaContainer}>{media}</div> : null}
```
because in case `media` is faulty (`0` for instance) `{media && <div className={styles.mediaContainer}>{media}</div>}` will return `0` and not `<div class="mediaContainer">0</div>`

Reply:
this ternary is still vulnerable to `0`

Reply:
Yes, my example wasn't that good.
But the difference is that with the ternary operator you decide what to return when it is falsy, and it is more readable.